package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "workspace-client",
+  "version": "0.0.1",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "typings/index.d.ts",
+  "scripts": {
+    "build": "rimraf dist && webpack-cli --mode production --config webpack.config.js",
+    "test": "jest",
+    "prepare": "npm run test && npm run build"
+  },
+  "contributors": [
+    {
+      "name": "Oleksii Kurinnyi",
+      "email": "okurinny@redhat.com"
+    }
+  ],
+  "license": "EPL-2",
+  "dependencies": {
+    "axios": "~0.18.0",
```

Comment:
we should avoid semver and stick to a given version (due to CQ dependencies)

yarn.lock is storing the correct version but it may be an issue

Reply:
done